src/common/ListItem.js

```diff
@@ -3,12 +3,14 @@
 import * as React from 'react';
 import { StyleSheet } from 'react-native';
 import { ListItem as RNEListItem } from 'react-native-elements';
+import { COLORS } from '../theme';
 
 const styles = StyleSheet.create({
-  // eslint-disable-next-line react-native/no-color-literals
   container: {
-    backgroundColor: 'white',
+    backgroundColor: COLORS.secondary,
```

Comment:
I think this should be a separate variable. It makes sense to set the font colors this way but I don't think it's good to set background color with the same variable.